pypika/tests/test_criterions.py

```diff
@@ -333,6 +332,45 @@ def test__function_isin(self):
         self.assertEqual('COALESCE("isin"."foo",0) IN (0,1)', str(c2))
 
 
+class IsNotInTests(unittest.TestCase):
+    t = Table('abc', alias='isnotin')
```

Comment:
@twheys Do you want me to change the alias of the other criterion tests as well? They are usually named after the criterion that is being tested.